velox/functions/lib/string/StringImpl.h

```diff
@@ -268,6 +268,15 @@ FOLLY_ALWAYS_INLINE bool md5_radix(
   return true;
 }
 
+/// Compute the SHA1 Hash.
+template <typename TOutString, typename TInString>
+FOLLY_ALWAYS_INLINE void sha1(TOutString& output, const TInString& input) {
```

Comment:
This code can be folded into the caller. No need to split the implementation into 2 files.

Reply:
Yes, sure. I can update. It looks that sha256 and sha512, and some of the other string functions have the split implementation. Just to confirm, would you like to follow the existing functions or should I go ahead and inline this code into the function itself?

Reply:
Initially, we used to split the implementation to allow for it to be used in the codegen solution, but we have not been maintaining codegen for a long time now, hence, no need to introduce more complexity then necesary.